invenio_collections/percolator.py

```diff
@@ -29,12 +31,16 @@ def new_collection_percolator(target):
     query = IQ(target.dbquery)
     for name in current_search.mappings.keys():
         if target.name and target.dbquery:
-            current_search.client.index(
-                index=name,
-                doc_type='.percolator',
-                id='collection-{}'.format(target.name),
-                body={'query': query.to_dict()}
-            )
+            try:
+                current_search.client.index(
+                    index=name,
+                    doc_type='.percolator',
+                    id='collection-{}'.format(target.name),
+                    body={'query': query.to_dict()}
+                )
+            except TransportError:
```

Comment:
(a) If you have less than 100+ collections then I would not use percolators.
(b) Have you tried https://www.elastic.co/guide/en/elasticsearch/reference/current/search-percolate.html#_forcing_unmapped_fields_to_be_handled_as_strings ?